CollAction/Models/ProjectViewModels/CommitProjectViewModel.cs

```diff
@@ -15,6 +15,6 @@ public class CommitProjectViewModel
 
         public bool IsActive { get; set; }
 
-        public string ProjectLink => $"/Projects/Details/{ProjectId}";
+        public string ProjectLink => $"/Projects/{ProjectName}/Details";
```

Comment:
This isn't used

Reply:
Yes but it is still in code so should be updated in case it is used in future, and we would need a story to cleanup unused things.

Reply:
Probably best to just clean it up. It's ok to clean stuff up related to the PR when you're working on it anyway, the functionality is tested anyway. As long as you limit the refactorings to the ticket.